tests/api/v1/samples/getWithLimit.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
I'm not using the default limit for the sample store. Should I be? The limit is applied after all the samples are retrieved so it won't help with performance.